ci_environment/bison/recipes/default.rb

```diff
@@ -25,24 +25,27 @@
 tmp = Dir.tmpdir
 case node[:platform]
 when "debian", "ubuntu"
-  path = File.join(tmp, node.bison.filename)
+   path = File.join(tmp, node.bison.filename)
 
-  remote_file(path) do
-    source node.bison.url
+   remote_file(path) do
+   source node.bison.url
 
-    owner node.travis_build_environment.user
-    group node.travis_build_environment.group
-  end
+     owner node.travis_build_environment.user
+     group node.travis_build_environment.group
+   end
 
-  file(path) do
-    action :nothing
-  end
+   file(path) do
+     action :nothing
+   end
 
-  package(path) do
-    action   :install
-    source   path
-    provider Chef::Provider::Package::Dpkg
+   package(path) do
+     action   :install
+     source   path
+     provider Chef::Provider::Package::Dpkg
 
-    notifies :delete, resources(:file => path)
+     notifies :delete, resources(:file => path)
+   end
+  package "bison" do
+    action [:install, :upgrade]
   end
```

Comment:
Lines 46-49 can be deleted.  I didn't delete all in the revert. 

the bison binary filename was fixed in attributes. 
https://github.com/travis-ci/travis-cookbooks/pull/842/commits/538831f10ecc36f642bdae383fb44e870635e026